### PR TITLE
Update Norwegian translation with new strings

### PR DESCRIPTION
**Pull request by zenseii**
This adds new strings to the NO translation and makes small corrections and adjustments to the translation.

### CODE REVIEW

File: files/lang/nb.po [hunk 1]
```diff
@@ -8802,6 +9299,42 @@ msgstr ""
 "https://github.com/ihhub/\n"
 "fheroes2/releases"
```
Comment (ihhub) [inline files/lang/nb.po:9301]: Please remove all lines below this line. They are not in use anymore.

Comment (zenseii) [reply files/lang/nb.po]: Done! I'd forgotten that unused lines are stored at the end...

**Comment by zenseii** [comment]
This PR is ready for review, although not having found any native speakers to review it I'm thinking I should go ahead and make  PR adding it to language options and another adding the specific letters to try to gather some interest.

**Comment by ihhub** [comment]
@zenseii , thanks a lot of taking care of translations!

**Pull request merged**